homeassistant/components/microbees/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "domain": "microbees",
+  "name": "microBees",
```

Comment:
All occurances of `microBees` in this PR and related PR need to be replaced by `Microbees`.

While it is indeed used with a capital letter `B` and lower case `m` in their logo, it is not part of their general branding. In all written communication, they spell it as "Microbees". 

```suggestion
  "name": "Microbees",
```

Reply:
Hello @frenck . The company name is microBees, also the Trade Mark is microBees (https://euipo.europa.eu/eSearch/#details/trademarks/013339593).  You can see in Logo, in Title in website. Of course we recognize ourself also with MicroBees and Microbees and in website you can find microBees as well MicroBees and Microbees.

Reply:
Cool, but however, we do follow how an upstream manufacturer brands itself.

Looking at the main website: https://www.microbees.com/

There is not a single occurrence of `microBees` in written text, nor is there any other occurrence you mentioned. All of them are consistently written as `Microbees`.

See:

![CleanShot 2024-02-16 at 18 00 29@2x](https://github.com/home-assistant/core/assets/195327/e5b14f2a-1f71-430c-b1f6-0a2850c0c394)

![CleanShot 2024-02-16 at 18 00 35@2x](https://github.com/home-assistant/core/assets/195327/f26987b6-2abb-466b-a228-40c1a259d830)

![CleanShot 2024-02-16 at 18 00 44@2x](https://github.com/home-assistant/core/assets/195327/321f06bb-75d9-497d-a587-f85de7108784)

![CleanShot 2024-02-16 at 18 00 52@2x](https://github.com/home-assistant/core/assets/195327/e1852e13-20d2-4fd2-a7e0-652fab212d16)

And more...

So, are you telling me your complete frontpage is incorrect?

../Frenck

Reply:
Hello @frenck as said this is not an issue for us because the company name is microBees but we use also MicroBees and Microbees as well. Anyway thank you for your suggestion, we've asked to our PR dep to use only microBees on the website to prevent confusion.

Reply:
But those are image brandings? As shown by your own website?

> the company name is microBees but we use also MicroBees and Microbees as well. 

Ok, can you point me to branding guidelines maybe to clear things up?

Reply:
This are shown on website. We've asked to change the Microbees occurrences on the website so prevent confusion. Trust me, i work for microBees from 10 years :) Our name is microBees.